etc/check_ipmi

```diff
@@ -27,6 +27,13 @@ if eeprom_key == 'ignore':
 try:
     allow_16bit = eeprom_key == '16-bit'
     image = eeprom.read_eeprom(allow_16bit)
+except OSError as e:
+    if e.errno == 6: # No FMC detected
```

Comment:
`errno == 6` means "no FMC detected"?  Really?!

I suppose I'd be slightly happier with the test `e.errno == errno.ENXIO`.  At least the manpage errno(3) tells us that this means:

> No such device or address (POSIX.1-2001).

which I guess is a pretty fair diagnosis.